code/game/gamemodes/wizard/artefact.dm

```diff
@@ -213,7 +213,7 @@
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
 	var/mob/living/carbon/human/target = null
 	var/list/mob/living/carbon/human/possible = list()
-	var/obj/item/link = null
+	var/obj/item/linked_item = null

```

Comment:
This is because `link` is also a byond built-in and accessing link procs causes compile errors unless you use src.link.